### DISCUSSION TITLE
Warp terminal

### DISCUSSION THREAD

**Discussion by t0msk** [catppuccin/catppuccin#2284] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### Any additional comments?

Please make catppucin theme for Warp terminal https://github.com/warpdotdev/Warp

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by nekowinston** [comment]
https://github.com/catppuccin/warp
